pipeline-scripts/buildlib.groovy

```diff
@@ -437,6 +452,105 @@ invalid mode build != master and no release branch
     return mode
 }
 
+//
+// Create a new version string based on the build mode
+//
+//@NonCPS
```

Comment:
You meant to leave the NonCPS commented out, right?

Reply:
I think I tested it with and without.  Groovy alone and Jenkins are different.  I'll verify.